creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -104,6 +104,19 @@ const healthCheckController = async (req) => {
     numberOfCPUs,
     randomBytesToSign
   )
+
+  const { stateMachineQueueLatestJobSuccess } = response
+  if (enforceStateMachineQueueHealth && stateMachineQueueLatestJobSuccess) {
+    const healthyThresholdMs = 5 * config.get('snapbackJobInterval')
```

Comment:
why is this 5?

Reply:
just picked a reasonable number, to allow for some instability - like we should only error after some amount. what do you think is reasonable @dmanjunath ?

Reply:
idk if we'd ever wire this up to a page, so i didnt feel a hard / soft alert was necessary

Reply:
reasonable. if 5 rounds are supposed to have passed and it hasn't, makes sense to at least wire up to a pingdom check

Reply:
yep exactly, cool